### PR TITLE
Modify molecule scenarios to test Oracle Linux 8

### PR DESCRIPTION
**Pull request by rrbadiani**
# Description
This PR aims to add molecule scenarios for Oracle Linux 8 support

1. Scenarios which use Oracle8 
1.1 archive-scram-rhel 
1.2 rbac-mtls-rhel8 
1.3 plain-customcerts-rhel-fips 
2. Installed the package tzdata-java in dockerfiles as it was missing in oracle8-java8 and oracle8-java11
3. Since same dockerfiles can work for Oracle8,Oracle9,Rhel8,Rhel9 removed extra separate dockerfiles.
4. Changed the docker image from redhat/ubi8 to redhat/ubi8-minimal in all scenarios to maintain uniformity, reduce the image size and use same dockerfiles across all rhel like images.
 

Fixes # [2478](https://confluentinc.atlassian.net/browse/ANSIENG-2478)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# How Has This Been Tested?

Oracle8 scenarios have been tested on the following ways - 

amd64 + zk     [2199](https://jenkins.confluent.io/job/cp-ansible-on-demand/2199/)  [2203](https://jenkins.confluent.io/job/cp-ansible-on-demand/2203/)
arm64 + zk      [2198](https://jenkins.confluent.io/job/cp-ansible-on-demand/2198/) [2202](https://jenkins.confluent.io/job/cp-ansible-on-demand/2202/) 
amd64 + kraft [2207](https://jenkins.confluent.io/job/cp-ansible-on-demand/2207/)
arm64 + kraft  [2208](https://jenkins.confluent.io/job/cp-ansible-on-demand/2208/)

# Checklist:

- [X] Any variable/code changes have been validated to be backwards compatible (doesn't break upgrade)
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] If required, I have ensured the changes can be discovered by cp-ansible discovery codebase
- [X] My code follows the style guidelines of this project
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: molecule/Dockerfile-rhel9-java8.j2 [hunk 1]
```diff
@@ -37,10 +37,8 @@ RUN yum -y install java-1.8.0-openjdk \
       krb5-workstation \
       unzip \
       procps \
-      procps-ng
-
-# Workaround to fix GPG signature validation issue, remove when it is fixed in CP 7.5 release
-RUN update-crypto-policies --set DEFAULT:SHA1
```
Comment (mansisinha) [inline molecule/Dockerfile-rhel9-java8.j2:43]: why did we remove this? Have we tested that fips+java8 works without this workaround?

**Comment by mansisinha** [comment]
Is it not possible to modify the dockerfiles which were used in rhel8 and use in oracle8 instead of using rhel9 files? It works with those files but is quite confusing and looks incorrect to use oracle8 image on rhel9 dockerfiles. 
If this is not possible, I believe it will be better to rename the rhel9 dockerfiles to something more generic if we're using the same files for oracle8 as well.

File: molecule/Dockerfile-rhel9-java8.j2 (hunk 1 above)
Comment (rrbadiani) [reply molecule/Dockerfile-rhel9-java8.j2:43]: In oracle 8 when we have this line it gives the error [jenkins](https://jenkins.confluent.io/job/cp-ansible-on-demand/2199/testReport/molecule.rbac-mtls-rhel8.molecule/yml/test/) Unknown policy SHA1. Removing it makes the scenario run fine. 
This combination of fips + java8 is not tested in any scenarios. I can modify one scenario to check this works fine without the SHA1 policy

**Comment by rrbadiani** [comment]
> Adding to the above comment, which image is using dockerfile-rhel-java* files now? Are those files even required if dockerfile-rhel9-java* files can work for both rhel8 and rhel9?

I think we dont need separate files for rhel8 and rhel9. They are having insignificant differences. we can use a generic dockerfiles for rhel8,rhel9,oracle8,oracle9

File: molecule/Dockerfile-rhel9-java8.j2 (hunk 1 above)
Comment (mansisinha) [reply molecule/Dockerfile-rhel9-java8.j2:43]: agreed, I think we don't even support fips on rhel9 with java8. It's only supported on java 11 and java 17. If removing this line works fine, I think it's good to remove

**Pull request merged**